<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc.qrc">
    <normaloff>:/img/1.png</normaloff>:/img/1.png</iconset>
  </property>
  <widget class="QWidget" name="widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>1111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>链表类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="listType">
      <item>
       <property name="text">
        <string>单链表</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>循环链表</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>双向链表</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="build_btn">
      <property name="text">
       <string>创建</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear_btn">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>1111</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>操作：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="operationType">
      <item>
       <property name="text">
        <string>插入</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>删除</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>修改</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>查找</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>index:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="index"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>val:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="val"/>
    </item>
    <item>
     <widget class="QPushButton" name="confirm_btn">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>1161</width>
     <height>521</height>
    </rect>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>451</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>631</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
